adobecom/milo
# MWPW-158674 [MEP] Support nested placholders in MEP placeholders

* Support using a MEP placeholder in a MEP placeholder.

Resolves: [MWPW-158674](https://jira.corp.adobe.com/browse/MWPW-158674)

Sample page tests:
1. the nested placeholder being before, after or in a different manifest than the one that uses it
2. a nested placeholder is defined twice and the last definition is used

QA instructions:
H1 should read: Buy now and save 50% off CC All Apps.
Description should start with: For just US$49.99,

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.page/drafts/mepdev/fragments/2024/q4/mepnestedplaceholders/?martech=off
- After: https://main--cc--adobecom.hlx.page/drafts/mepdev/fragments/2024/q4/mepnestedplaceholders/?milolibs=mepnestedplaceholders&martech=off
- Psi-check: https://mepnestedplaceholders--milo--adobecom.hlx.page/?martech=off

Also includes https://github.com/adobecom/milo/pull/2919 for conflict resolution
